src/text-tags.js

```diff
@@ -42,6 +42,22 @@ function buildTagsFromStory(config, story, url = {}) {
   return storyMetaData;
 }
 
+function buildTagsFromTopic(config, data, url = {}) {
+
+  const topicUrl = `${config['sketches-host']}/topic/${data.tag}`;
```

Comment:
topic urls are generated from the front end app. might differ based on publisher. we might have to do handle this similar to section.

Reply:
done